README.md

```diff
@@ -1 +1,14 @@
-# DX_UPD
\ No newline at end of file
+# DX_UPD
+
+All script are made tested using Python 3.6.8
+
+## Installation
+To run the UPD scripts we need to create a virtual python environment
+
+### Make virtual python environment
+```bash
+python3 -m venv venv
+source venv/bin/activate
```

Comment:
would it be beneficial to always confirm you’re in the virtual environment by checking the location of your Python by adding this command: "which python"? 
OR
you can also skip the "activation" and instead directly call the executables that are in the virtual environment's bin directory, for example path/to/venv/bin/python something.py